tests/test_ext_napoleon.py

```diff
@@ -129,16 +135,17 @@ def assertSkip(self, what, member, obj, expect_default_skip, config_name):
             self.assertEqual(None, _skip_member(app, what, member, obj, skip,
                                                 mock.Mock()))
         else:
-            self.assertFalse(_skip_member(app, what, member, obj, skip,
-                                          mock.Mock()))
+            self.assertIs(_skip_member(app, what, member, obj, skip,
+                                       mock.Mock()), False)
         setattr(app.config, config_name, False)
         self.assertEqual(None, _skip_member(app, what, member, obj, skip,
                                             mock.Mock()))
 
-    def test_namedtuple(self):
-        self.assertSkip('class', '_asdict',
-                        SampleNamedTuple._asdict, False,
-                        'napoleon_include_private_with_doc')
+    # TODO: this test has been failing for a while and should be fixed.
+    # def test_namedtuple(self):
+    #     self.assertSkip('class', '_asdict',
+    #                     SampleNamedTuple._asdict, False,
```

Comment:
Please set `expect_default_skip=True` here?
It seems nothing changed for this code from its beginning (bbfd0d058aecf85bd3b711a846c83e2fe00fa136).

Reply:
The current behavior is `None`. I have changed `expect_default_skip` to `None` so now the tests should pass.

Reply:
Oh you're right, it seems `True` is the correct value here

Reply:
Thank you for update. I noticed this is broken by difference of python version.
```
$ python3.7
Python 3.7.8 (default, Jun 29 2020, 01:48:49)
[Clang 11.0.3 (clang-1103.0.32.59)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from collections import namedtuple
>>> SampleNamedTuple = namedtuple('SampleNamedTuple', 'user_id block_type def_id')
>>> 'SampleNamedTuple' in SampleNamedTuple._asdict.__globals__
False
```
```
$ python3.5
Python 3.5.8 (default, Mar  1 2020, 23:40:05)
[GCC 4.2.1 Compatible Apple LLVM 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from collections import namedtuple
>>> SampleNamedTuple = namedtuple('SampleNamedTuple', 'user_id block_type def_id')
>>> 'SampleNamedTuple' in SampleNamedTuple._asdict.__globals__
True
```

As you commented originally, some work is needed for this. I don't know what is happened in a real document. But I consider this is not a blocker of your PR. So please comment-out this again as you first post. I'll work for it in another PR. Sorry for confusing you.

Reply:
I fixed this testcase in #7920. So merging HEAD of 3.x branch will be fixed it.